drivers/dai/intel/ssp/ssp.c

```diff
@@ -706,7 +706,7 @@ static inline void dai_ssp_pm_runtime_en_ssp_clk_gating(struct dai_intel_ssp *dp
 
 static void dai_ssp_pm_runtime_en_ssp_power(struct dai_intel_ssp *dp, uint32_t index)
 {
-#if CONFIG_SOC_SERIES_INTEL_CAVS_V25
+#if CONFIG_SOC_SERIES_INTEL_CAVS_V25 || !CONFIG_INTEL_ADSP_CAVS
```

Comment:
FWIW: this kind of platform test is generally best expressed with things like a devicetree property or a CONFIG_HAS_* kconfig, etc...